day_7/ceasar_cipher.md

```diff
@@ -14,3 +14,8 @@ cipher = CeasarCipher.new
 cipher.encode("Hello World", 5)
 => "CZGGJ RJMGY"
 ```
+prompt user for code and how many steps to right or left
+
+turns the string into an array of numbers
+make alphabet array
```

Comment:
We were looking to see one of either the cipher or the checkerboard coded out, as per the instructions. We aren't expecting perfection here, but these challenges are great ways to practice breaking problems apart, like you've started here, and implementing some research into how to achieve those steps. I'm seeing enough from your class work, speckled frogs, and fizzbuzz to know you're on the right track as far as material we want students to enter Turing with, but I highly encourage trying out BOTH the cipher and the checkerboard in the weeks leading up to your start date as excellent practice. Feel free to drop me a line via slack if you do add to your work here, and would like feedback on that.

Reply:
I looked at the repo again, and I see coding these out is optional (this is a change). Regardless, I encourage you to give them a shot! Just wanted to reply here that I was mistaken that coding them out was asked of you.

Reply:
Mason, thanks for the feedback. I submitted my caesar cipher and would love some feedback. 

a few questions i had:
-should i make it able to accept phrases as well?
-is there a way to refactor the code so it's more readable, neat, and has better separation of functionality? I still get a little confused as to where i need to define things in the class.
-are there any important tools or methods i'm missing that might make this easier? 

i'm starting to work on the checkerboard now.